I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 35, in script
    "Another kid went missing? Didn't some guy go missing too yesterday? And the day before?"
  File "renpy/common/000window.rpy", line 114, in _window_auto_callback
    _window_show(auto=True)
  File "renpy/common/000window.rpy", line 69, in _window_show
    renpy.with_statement(trans)
OSError: Couldn't find file 'audio/bensound-onceagain.mp3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 35, in script
    "Another kid went missing? Didn't some guy go missing too yesterday? And the day before?"
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/ast.py", line 891, in execute
    statement_name("say")
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/ast.py", line 47, in statement_name
    i(name)
  File "renpy/common/000window.rpy", line 114, in _window_auto_callback
    _window_show(auto=True)
  File "renpy/common/000window.rpy", line 69, in _window_show
    renpy.with_statement(trans)
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/exports.py", line 1657, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/display/core.py", line 2798, in do_with
    return self.interact(trans_pause=True,
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/display/core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/display/core.py", line 4103, in interact_core
    renpy.audio.audio.periodic()
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/audio/audio.py", line 1166, in periodic
    raise_(exc[0], exc[1], exc[2])
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/audio/audio.py", line 1144, in periodic_thread_main
    periodic_pass()
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/audio/audio.py", line 1086, in periodic_pass
    c.periodic()
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/audio/audio.py", line 517, in periodic
    topf = load(filename)
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/audio/audio.py", line 70, in load
    rv = renpy.loader.load(fn)
  File "/Users/yourkucing/Documents/renpy-8.0.3-sdk/renpy/loader.py", line 820, in load
    raise IOError("Couldn't find file '%s'." % name)
OSError: Couldn't find file 'audio/bensound-onceagain.mp3'.

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.0.3.22090809
The Couch 1.0
Wed Jan  3 21:27:40 2024
